module-about/src/main/java/com/orange/ods/module/about/ui/navigation/OdsAboutNavigation.kt

```diff
@@ -37,20 +37,24 @@ import com.orange.ods.module.about.ui.configuration.OdsAboutUrlMenuItem
 import com.orange.ods.module.about.ui.utilities.extension.launchUrl
 
 
-private object OdsAboutDestinations {
+object OdsAboutDestinations {
     const val HomeRoute = "ods/module/about/home"
-    const val AboutRoute = "ods/module/about/"
-    const val FileItemRoute = "ods/module/about/fileItem"
-    const val AppNewsRoute = "ods/module/about/appNews"
-    const val AccessibilityStatementRoute = "ods/module/about/accessibilityStatement"
+    internal const val AboutRoute = "ods/module/about/"
```

Comment:
Shouldn’t this route be public instead of `HomeRoute` because this is the entry point of the nested navgraph?

Reply:
Yes that's what I did at first, but then I had issues with the top app bar title. This is due to the fact that when we navigate to `AboutRoute`, the current route becomes `HomeRoute` because it is the `startDestination` of the nested graph.

I thought we had to navigate to `AboutRoute` to display the about screen but it seems to work perfectly with `HomeRoute`. That's why I chose to only make `HomeRoute` public instead.

Another solution would be to have several methods:

```kotlin
fun NavController.navigateToOdsAbout(navOptions: NavOptions? = null) {
    navigate(OdsAboutDestinations.AboutRoute, navOptions)
}
```

and:

```kotlinfun NavController.isCurrentDestinationAbout(): Boolean {
    return currentDestination?.route == OdsAboutDestinations.HomeRoute
}
```

But I am not sure i's better.

Reply:
I think this is not a good practice to access directly to the `startDestination`. I found a way to manage nested graph with a  bottom bar in the official documentation [here](https://developer.android.com/develop/ui/compose/navigation#bottom-nav). So I think we should expose the `AboutRoute` and modify bottom bar management in the demo app by using the following line for selected item:

```kotlin
selected = currentDestination?.hierarchy?.any { it.route == item.route } == true,
```